noxfile.py

```diff
@@ -249,20 +249,20 @@ def bump(session: nox.Session):
         "tag",
         "-a",
         "-m",
-        f"antsibull-changelog {version}",
+        f"antsibull-docs-parser {version}",
```

Comment:
This seems like a copy paste error :)

Reply:
More like a "saved the wrong editor window" bug, I fixed this copy'n'paste error earlier on `main` directly, but the editor window where I making the other changes apparently had an older version...